stable/rabbitmq/README.md

```diff
@@ -88,8 +88,8 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `ingress.annotations`                | ingress annotations as an array                  | []                                                      |
 | `livenessProbe.enabled`              | would you like a livenessProbed to be enabled    | `true`                                                  |
 | `livenessProbe.initialDelaySeconds`  | number of seconds                                | 120                                                     |
-| `livenessProbe.timeoutSeconds`       | number of seconds                                | 5                                                       |
-| `livenessProbe.periodSeconds`        | number of seconds                                | 5                                                       |
+| `livenessProbe.timeoutSeconds`       | number of seconds                                | 20                                                      |
+| `livenessProbe.periodSeconds`        | number of seconds                                | 30                                                      |
```

Comment:
doesn't it miss the default values for readiness probes?

Reply:
You're right. Check this new PR: https://github.com/helm/charts/pull/10380